data/lang/module-scout/en.json

```diff
@@ -0,0 +1,482 @@
+  },
+  "ADTEXT_7_WHYSOMUCH" : {
+    "description" : "",
+    "message" : "Time is money."
+  },
+  "ADTITLE_8": {
+    "description": "",
+    "message": "CARTOGRAPHY ORDER"
+  },
+  "ADTEXT_8" : {
+    "description" : "",
+    "message" : "Update of data for {systembody} needed"
+  },
+  "ADTEXT_8_INTRO" : {
+    "description" : "",
+    "message" : "Hi! \nI'm with the geographical society, and we need to update our maps of {systembody}. We are willing to pay {cash} for a pilot willing to fly lower than our previous."
+  },
+  "ADTEXT_8_SUCCESSMSG" : {
+    "description" : "",
+    "message" : "Another map to the collection. Please find our appreciation reflected in your bank account."
+  },
+  "ADTEXT_8_FAILMSG" : {
+    "description" : "",
+    "message" : "You fail to appreciate the fine art of maps, and timely delivery of agreed upon date!"
+  },
+  "ADTEXT_8_WHYSOMUCH" : {
+    "description" : "",
+    "message" : "The worlds must be documented and mapped, in ever finer and greater detail."
+  },
+  "ADTITLE_9": {
+    "description": "",
+    "message": "MAPPING"
+  },
+  "ADTEXT_9" : {
+    "description" : "",
+    "message" : "Will pay for fast delivery of data on {systembody} in {system}"
+  },
+  "ADTEXT_9_INTRO" : {
+    "description" : "",
+    "message" : "Hi! \nI'm {name}. Me and my family need a detailed survey of {systembody} in {system}, before anyone else claims it. We will pay {cash} for a fast pilot. We'll message you with the delivery location once you have the data."
+  },
+  "ADTEXT_9_SUCCESSMSG" : {
+    "description" : "",
+    "message" : "Your punctuality and discretion is appreciated."
+  },
+  "ADTEXT_9_FAILMSG" : {
+    "description" : "",
+    "message" : "You're late! Now this is of no use to us, since it has already been claimed. Another opportunity lost. Like tears in rain."
+  },
+  "ADTEXT_9_WHYSOMUCH" : {
+    "description" : "'scooped' as in missing an exclusive unique opportunity, before someone else gets to it first",
+    "message" : "We do not want to be scooped on this possible acquisition"
+  },
+  "MAPPING_COMPLETED" : {
+    "description" : "",
+    "message" : "MAPPING COMPLETED. Data saved."
+  },
+  "DANGER" : {
+    "description" : "",
+    "message" : "Danger:"
+  },
+  "DEADLINE" : {
+    "description" : "",
+    "message" : "Deadline:"
+  },
+  "DISTANCE" : {
+    "description" : "",
+    "message" : "Distance:"
+  },
+  "DISTANCE_REACHED_ORBITAL" : {
+    "description" : "",
+    "message" : "DISTANCE REACHED. MAPPING. Remain within orbit parameters until scan completed."
+  },
+  "DISTANCE_REACHED_SURFACE" : {
+    "description" : "",
+    "message" : "DISTANCE REACHED. MAPPING. Remain below altitude ceiling until scan completed."
+  },
+  "EXCELLENT_I_AWAIT_YOUR_REPORT" : {
+    "description" : "",
+    "message" : "Excellent. I will await your report."
+  },
+  "HOW_DOES_IT_WORK" : {
+    "description" : "",
+    "message" : "How does it work."
```

Comment:
Does this not need a question mark?

Reply:
Yes, I've added it now, hopefully before most translators started working on it.